src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/leaked-passwords/leakedPasswordsData.tsx

```diff
@@ -286,7 +303,10 @@ function getLeakedPasswords(props: LeakedPasswordLayout) {
     },
   ];
 
-  return leakedPasswordsData.find((content) => content.type === dataType);
+  return {
+    breachName,
```

Comment:
It's not clear to me which breach this would be the `breachName` of, but it doesn't seem to be used anywhere anyway? Maybe this should be reverted to its old return value?

Reply:
OK, cool. For the final parameter name, it might be good if it indicates which breach it's referring to - as I interpret the function name, `getLeakedPasswords` returns leaked passwords across all breaches.